MuteSpotifyAds/AppDelegate.swift

```diff
@@ -13,13 +13,15 @@ import Foundation
 class AppDelegate: NSObject, NSApplicationDelegate, NSUserNotificationCenterDelegate {
     let endlessPrivateSessionKey = "EndlessPrivateSession"
     let restartToSkipAdsKey = "RestartToSkipAds"
+    let startSpotifyAtAppStartKey = "StartSpotifyWhenMSAStarts"
```

Comment:
Can you please rename the option to simply `startSpotify`, since else the name gets very long and leads to inconsistent abbreviations like `startSpotifyATS`.

Reply:
The key then should be `"StartSpotify"`

Reply:
Yes